BoxContentSDK/BoxContentSDK/Operations/BOXAPIMultipartToJSONOperation.m

```diff
@@ -590,6 +587,11 @@ - (void)writeDataToOutputStream
 {
     while ([self.outputStream hasSpaceAvailable])
     {
+        if (self.isCancelled) {
```

Comment:
These changes are both in one method. I believe there's only one stream delegate method for handling events in the runloop (handleEvent) so I don't think we need to worry about some other method.